tests/lib/rules/attributes-order.js

```diff
@@ -1246,6 +1282,50 @@ tester.run('attributes-order', rule, {
         'Attribute "v-if" should go before "v-on:click".'
       ]
     },
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <div
+          v-custom-directive="x"
+          v-bind="b"
+          v-model="c"
+          v-bind:value="a">
+        </div>
+      </template>`,
+      output: `
+      <template>
+        <div
+          v-bind="b"
+          v-model="c"
+          v-custom-directive="x"
+          v-bind:value="a">
+        </div>
+      </template>`,
+      errors: ['Attribute "v-model" should go before "v-custom-directive".']
+    },
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <div
+          v-if="x"
+          v-model="c"
+          v-bind="b"
+          v-bind:id="a">
+        </div>
+      </template>`,
+      output: `
+      <template>
+        <div
+          v-if="x"
+          v-bind="b"
+          v-bind:id="a"
+          v-model="c">
+        </div>
+      </template>`,
+      errors: ['Attribute "v-bind:id" should go before "v-model".']
+    },
```

Comment:
Will this create false positives in existing Vue 2 code? There, the order `v-if, v-model, v-bind, v-bind:id` would be perfectly fine.

Reply:
I don't think there are any false positives because the `id` must be before the `v-model`.

https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended

Reply:
Oh, indeed.

Reply:
Oh, indeed.